src/Debug/skeletonViewer.ts

```diff
@@ -735,6 +742,48 @@ export class SkeletonViewer {
         }
     }
 
+    private _buildLocalAxes(): void {
+        if (this._localAxes) {
+            for (let axisMesh of this._localAxes) {
+                axisMesh.dispose();
+            }
+
+            this._localAxes = [];
+        }
+
+        let displayOptions = this.options.displayOptions || {};
+
+        if (!displayOptions.showLocalAxes) {
+            return;
+        }
+        
+        if (this._utilityLayer) {
+            const targetScene = this._utilityLayer.utilityLayerScene;
+            const size = displayOptions.localAxesSize || 0.075;
+
+            if (targetScene) {
+                for (let b of this.skeleton.bones) {

```

Comment:
You can avoid some testings and indentation levels in this function because `this._localAxes`, `this.options.displayOptions`, `this._utilityLayer` and `targetScene` will always be non null and defined.